test/spec/adapters/rubicon_spec.js

```diff
@@ -0,0 +1,292 @@
+import { expect } from "chai";
+import adloader from "src/adloader";
+import adapterManager from "src/adaptermanager";
+import bidManager from "src/bidmanager";
+
+var CONSTANTS = require("src/constants.json");
+
```

Comment:
`; and describe should have at most 2 line(s) between them at test/spec/adapters/rubicon_spec.js :

      4 |    import bidManager from "src/bidmanager";
      5 |    
      6 |    var CONSTANTS = require("src/constants.json");
--------------------------------------------------------^
      9 |
    10 |    describe("the rubicon adapter", () => {

1 code style error found.`